smac/tae/execute_func.py

```diff
@@ -170,7 +187,17 @@ def run(self, config, instance=None,
                 cost = self.crash_cost
             
             runtime = time.time() - start_time
-            additional_run_info = {}
+
+        # check serializability of results
+        try:
+            json.dumps(cost)
+        except TypeError:
+            raise TypeError("TA returned result of type %s but it should be a serializable type." % type(cost))
```

Comment:
It would be great if you could

1. print the actual return value as well
2. explicitly note that it must be a type serializable by json and ask the user to convert it.